Corona-Warn-App/src/main/res/layout/statistics_trend_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This additional nested `FrameLayout` is not necessary, because the custom view is a `FrameLayout` already.

Reply:
True, I removed it